packages/mux-audio/src/index.ts

```diff
@@ -254,7 +248,22 @@ class MuxAudioElement extends CustomAudioElement<HTMLAudioElement> implements Pa
   }
 
   get metadata() {
-    return this.#metadata;
+    const inferredMetadataAttrs: { [key: string]: string } = {};
+    this.getAttributeNames()
+      .filter((attrName) => {
+        return attrName.startsWith('metadata-') && !([Attributes.METADATA_URL] as string[]).includes(attrName);
+      })
+      .map((attrName) => {
```

Comment:
nitpick(blocking): either use `forEach` if you're producing a side effect (like your implementation in `mux-player`) or refactor to use `reduce()`, e.g.:

```ts
const inferredMetadataAttrs: { [key: string]: string } = this.getAttributeNames()
      .filter((attrName) => {
        return attrName.startsWith('metadata-') && !([Attributes.METADATA_URL] as string[]).includes(attrName);
      })
      .reduce((curAttrs, attrName) => {
        const value = this.getAttribute(attrName);
        if (value != null) {
          curAttrs[attrName.replace(/^metadata-/, '').replace(/-/g, '_') as string] = value;
        }
        return currAttrs;
      }, {});
```

Reply:
good call!